oshi-core/src/main/java/oshi/util/IPUtil.java

```diff
@@ -0,0 +1,259 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2010 - 2021 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package oshi.util;
+
+import inet.ipaddr.IPAddress;
```

Comment:
why not use java built-in `java.net.InetAddress`

Reply:
Reviewing the util I think the main purpose of this is shortening IPv6 addresses.  There are [many ways to do this](https://stackoverflow.com/questions/7043983/ipv6-address-into-compressed-form-in-java) without a separate dependency... I've done so myself in a different project.

Reply:
In fact I did so in this project at some point using a regex, and it's in `ParseUtil.parseUtAddrV6toIP()`.  That should/could be refactored out into a public helper method if needed, the rest of the functions from this dependency aren't needed.